pom.xml

```diff
@@ -31,16 +31,14 @@
     <parent>
         <groupId>io.gravitee</groupId>
         <artifactId>gravitee-parent</artifactId>
-        <version>22.1.1</version>
+        <version>23.0.2</version>
     </parent>
 
     <properties>
-        <gravitee-bom.version>8.1.0</gravitee-bom.version>
-        <gravitee-apim.version>4.3.7</gravitee-apim.version>
-        <gravitee-gateway-api.version>3.5.0</gravitee-gateway-api.version>
-        <gravitee-policy-api.version>1.11.0</gravitee-policy-api.version>
-        <gravitee-common.version>4.4.0</gravitee-common.version>
-        <aws-java-sdk.version>1.12.748</aws-java-sdk.version>
+        <gravitee-apim-bom.version>4.7.0-alpha.1</gravitee-apim-bom.version>
+        <gravitee-entrypoint-http-get.version>2.1.0</gravitee-entrypoint-http-get.version>
+        <gravitee-reactor-message.version>6.0.0</gravitee-reactor-message.version>
+        <aws-java-sdk.version>2.30.26</aws-java-sdk.version>
 
         <maven-plugin-assembly.version>3.5.0</maven-plugin-assembly.version>
         <maven-plugin-prettier.version>0.19</maven-plugin-prettier.version>
```

Comment:
useless it is defined in parent. Could you clean all prettier mention (props & plugins)